### ISSUE TITLE
Demo misrepresents functionality

### ISSUE THREAD

**Issue by 21stcn** [tareq1988/wedocs#21]
I really appreciate the effort you've taken in creating this theme and sharing it, however the demo site is misleading. I cannot replicate it using the theme and the documentation provided in the readme. Specifically, the sidebar navigation is vastly different from the demo. In the theme, h1 tags in a specific post have menu items in the sidenav, but there are no sidenav menu items for specific posts within categories as in the demo. 

I would suggest you set up a representative demo site or update the documentation (readme) to explain how this functionality can be achieved.

The benefit of this amazing theme you have created is that developers can quickly and easily set up accessible documentation with good navigation. Unfortunately because the demo is different from the actual theme, you end up costing people time who are misled. While I appreciate that a developer can easily branch off and fix things up, this undermines the "quick and easy" part and essentially this theme is no longer suitable for many people who would like to plug and play and replicate the demo without hassle.

As others have mentioned, people would gladly pay for a premium version if it works as demonstrated, but currently it seems to be costing a few people time they could have saved buying a premium theme. Thanks again.

**Comment by BigIron5** [comment]
@21stcn and @tareq1988 I can't agree with this more. I spent many hours (almost an entire day) trying to figure out why I couldn't get this to behave like the demo. You have a great system here that people want, and I suspect it wouldn't take you long to deliver.

**Comment by tareq1988** [comment]
Yes, the current site you guys are seeing is different from the theme here. I've reworked with the theme and made that new navigation and the code isn't here. This is how the default weDocs theme looks like: http://support.daffyhazan.com/online-docs/installation-4/

The reworked version removed the navigation menu and added static menus, introduced custom post types for documentation. Few other design changes, I was busy and a lot of things going on, so couldn't update. So don't get confused by the current documentation site of [weDevs](http://docs.wedevs.com)

**Comment by BigIron5** [comment]
@tareq1988 Thank you for the reply. 

Any chance you'd be willing to share the current version? That's exactly what I (and it sounds like others) are looking for.

**Comment by planetahuevo** [comment]
Hi.
I was hoping to find the demo version and It really different of the actual code. 
Is it possible that you share the current version.

Thank you very much for this theme, it is great work too. 

Thanks

**Comment by planetahuevo** [comment]
I manage to create the sidebar. Thanks

**Comment by soulpimp** [comment]
@planetahuevo how did you get the sidebar to work?

**Comment by planetahuevo** [comment]
@soulpimp well, doing two things, first updating the code myself a little bit to make it work with H2 instead of H1 (I plan to release an updated code with some more changes, but i need to clean the code a little bit and finish some stuff).

For making it work with the actual code: 

1. Create a new post. 
2. Fill it up with plain content.
3. Mark come lines with H1 and only with H1 tags. this is important. Only clean h1 tags will work.
4. Publish. 
5. Visit the page, if everything works fine, the H1 will appear on the sidebar.

**Comment by bohotina** [comment]
Am I missing something with this theme? I download the files, uploaded them to the themes forlder under WP. When I go to select the theme it doesn't even show up. 

Any suggestions?

**Comment by planetahuevo** [comment]
I think we are not using this issues properly... You should open a new issue.

**Comment by BigIron5** [comment]
@planetahuevo I'd greatly appreciate you releasing the updated code so that this functions the same as the demo here: http://docs.wedevs.com/docs/wp-user-frontend-pro/getting-started/creating-posting-forms/

**Comment by planetahuevo** [comment]
@BigIron5 That demo is outdated as you can read the comment by darek. 
I have been traveling all week and were not able to do any coding. I will try to upload the changes this weekend but I cannot promise it.

**Comment by BigIron5** [comment]
@planetahuevo No problem, really looking forward to it. If I'm able to get it working just like the demo I'll gladly make a donation to your beer fund ;)

**Comment by tareq1988** [comment]
Updated the repo with the latest version. Now the theme uses a `docs` post type instead of `post` post type.

You can use [this plugin](https://wordpress.org/plugins/convert-post-types/) to bulk convert the post types from one to another.

To get this kind of [sidebar navigation](http://docs.wedevs.com/docs/wp-project-manager/getting-started/view-reports/):
<img width="570" alt="screen shot 2016-02-10 at 11 57 56 pm" src="https://cloud.githubusercontent.com/assets/153669/12956432/39ae7264-d052-11e5-8880-485838bbe3e9.png">

You have to arrange the docs post in parent-child relationship
<img width="617" alt="screen shot 2016-02-10 at 11 58 59 pm" src="https://cloud.githubusercontent.com/assets/153669/12956461/5c1d2480-d052-11e5-8ee5-a3f1de35f680.png">

The initial version was different than the [current weDevs Documentation](http://docs.wedevs.com/). Both [versions are tagged](https://github.com/tareq1988/wedocs/releases), so you can use any one of them.

**Comment by BigIron5** [comment]
@tareq1988 That's great! 

<edit: found>

**Comment by planetahuevo** [comment]
@tareq1988 thank you! I will keep my work on my fork (not public yet sorry) and I am trying to get some ideas from the new one. :+1:  But it will take some time.  
@BigIron5 no beers will be needed but thank you!!

**Comment by frumbert** [comment]
Is there a way to generate the front page of the site to look like the example image in this plugins description? E.g. with categorised boxes for each document that actually link to the sections within that document structure, etc.
